cmd/gaia/app/genesis.go

```diff
@@ -20,6 +22,8 @@ var (
 	// bonded tokens given to genesis validators/accounts
 	freeFermionVal  = int64(100)
 	freeFermionsAcc = int64(50)
+
+	DefaultKeyPass = "12345678"
```

Comment:
const?

Reply:
also why doesn't this go to 9? I though the default pass has been `1234567890` for a while (nice for whipping your hands across the top row of the keyboard.

Reply:
I think 12345678 is more common for people to try since we only require 8 chars

Reply:
I'll make it a `const` 👍

Reply:
cool